overlord/state/change.go

```diff
@@ -23,6 +23,17 @@ import (
 	"encoding/json"
 )
 
+// Status is used for status values for changes and tasks.
+type Status int
+
+// Admitted status values for changes and tasks.
+const (
+	Running Status = 0
```

Comment:
perhaps the perfect spot to use iota unless we specifically want to use particular values forever across re-shuffling of those constants.